pagopa/io-app
# chore: [PE-535] Adapted the FooterWithButtons into CGN screens

## Short description
This PR adapts the CGN screens with the new `FooterWithButtons` released into the new version of the design system `v1.21.0`

## List of changes proposed in this pull request
- Removed the `SafeAreaView` from CGN screens with a `FooterWithButtons` inside

## How to test
- Run all the CGN flows such as the activation and details, you should be able to see the right footer with buttons using the right margin.

## Preview
|Before|After|
|-|-|
| <img src="https://github.com/pagopa/io-app/assets/34343582/3e7fcefd-6c9e-4fc4-b595-98068cc382ff" width="300" /> | <img src="https://github.com/pagopa/io-app/assets/34343582/c334e2ce-8997-49d1-8e7a-3fabf2b7ac6a" width="300" /> |

## Acknowledgment
@dmnplb
